script/1_0_2current_en_script/Arcueid/Day 7/07_00_ARC07_1B.txt

```diff
@@ -1,82 +1,82 @@
 <Page5>
 I was also invited to club activities back when I was a freshman.
 Whenever that happened, I always turned them down by saying 'maybe next time'.
 Each time that I did that, I felt an indescribable sense of distance between us.
 <Page6>
 ...In the end, that's what happened.
-I'm sure it was part of my self-defense mechanism telling me unconciously to not get involved with them under any circumstance.
+I'm sure it was part of my self-defense mechanism unconsciously telling me not to get involved with them under any circumstances.
 <Page7>
 "..."
 I put an end to my boring sentiments.
-I send my support to the hard-working club members from the bottom of my heart and walk toward the school building.
+I cheer on the hard-working club members from the bottom of my heart and walk toward the school building.
 <Page8>
 "O-Oh my. Good morning Shiki-kun, aren't you a bit early this morning?"
 "...Yes, I got the time a bit mixed up.
 Good morning, Noel-sensei."
 <Page9>
 On my way to the classroom, I come across Noel-sensei unexpectedly.
-Just like me, Noel-sensei didn't expect anyone to be here at such an hour and looks a little dumb-founded.
+Just like me, it seems Noel-sensei didn't expect anyone to be here at such an hour and looks a little dumb-founded.
 <Page10>
-"Rather than getting the time mixed up and coming to school late, you came to school early instead. You must be one of those types, the hard working ones right? But while you're still young, wouldn't it be better to live a bit more relaxed?"
-"But I'm more than relaxed. Besides, I don't even like to study.
-More importantly, I want to talk about my truancy from last Friday..."
+"Rather than getting the time mixed up and coming to school late, you came to school early instead. You must be one of those types, the hard working ones right? But while you're still young, wouldn't it be better to loosen up a little bit?"
+"I'm plenty laid-back. Besides, I don't even like to study.
+More importantly, I want to talk about my unnotified absence from last Friday..."
 <Page11>
-"Oh, you mean that. Don't worry, I didn't contact your home.
-It's totally normal to boycott every once in a while. It would have been more convenient for you this way, right Shiki-kun?"
+"Oh, you mean that. Don't worry, I didn't contact your family.
+It's totally normal to boycott every once in a while. Besides, it's more convenient for you this way, right Shiki-kun?"
 <Page12>
-Noel-sensei gives off a chuckle with hidden meaning and makes a statement that is not appropriate for a teacher.
+Noel-sensei gives off a suggestive chuckle and makes a statement inappropriate for a teacher.
 ...I guess that helps.
 Or rather, it did help. The only reason Akiha wasn't so overly angry was because she didn't know about my absence on Friday.
 "――――――"
-That being said, isn't Noel-sensei's educational policy a bit too generous?
+That being said, isn't Noel-sensei's educational policy a bit too lenient?
```

Comment:
Too J-E IMO
> とはいえ、ノエル先生の教育方針は、ちょっと<大|おお>らかすぎないだろうか？
I get the feeling that Noel-sensei's education policy is just a tad too lenient.